pkg/cloudprovider/provider/gce/cloudconfig_test.go

```diff
@@ -52,7 +53,9 @@ func TestCloudConfigAsString(t *testing.T) {
 				"subnetwork-name = \"my-cool-subnetwork\"\n" +
 				"token-url = \"nil\"\n" +
 				"multizone = true\n" +
-				"regional = true\n",
+				"regional = true\n" +
+				"node-tags[] = \"tag1\"\n" +
```

Comment:
https://www.npmjs.com/package/ini#usage has a good example for arrays :)

Reply:
Ini is not standardized, do we have other occurences of slices? If not, could you verify that this actually works with the ini parser that is being used in k/k?

Reply:
Updated. Ref: `Example (Multivalue)` here - https://godoc.org/gopkg.in/gcfg.v1

```
package main

import (
	"fmt"
	gcfg "gopkg.in/gcfg.v1"
	"log"
)

func main() {
	cfgStr := `; Comment line
[section]
nodeTags="tag1"
nodeTags="tag2"`

	cfg := struct {
		Section struct {
			NodeTags []string
		}
	}{}
	err := gcfg.ReadStringInto(&cfg, cfgStr)
	if err != nil {
		log.Fatalf("Failed to parse gcfg data: %s", err)
	}
	fmt.Println(cfg.Section.NodeTags)
}
```

gives

```
[tag1 tag2]
```

Reply:
/lgtm
/approve